build.gradle

```diff
@@ -30,6 +30,7 @@ ext {
 		gson_version                       : "2.8.9",
 		httpcomponents_core_version        : "4.4.16",
 		httpcomponents_client_version      : "4.5.14",
+		passay_version					   : "1.6.4",
```

Comment:
Minor but can we fix the indenting here so it matches the above

Reply:
What is odd is that it's in place correctly on IntelliJ so not sure how this happened:
<img width="649" alt="Screenshot 2023-12-13 at 11 31 04" src="https://github.com/govuk-one-login/ipv-cri-dl-api/assets/142988564/e8204658-cdb6-4e00-81ed-c8ab3fa6a65f">

Reply:
Fixed it